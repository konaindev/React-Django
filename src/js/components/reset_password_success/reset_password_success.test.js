import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";

import ResetPasswordSuccess from "./index";

describe("ResetPasswordSuccess", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <ResetPasswordSuccess></ResetPasswordSuccess>
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
