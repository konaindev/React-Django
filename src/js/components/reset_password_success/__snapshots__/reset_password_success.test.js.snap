// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordSuccess renders correctly 1`] = `
<div
  className="page-auth"
>
  <div
    className="page-auth__header"
  >
    <a
      className="page-auth__back-link"
      href="/"
      onClick={[Function]}
      style={
        Object {
          "color": "inherit",
          "textDecoration": "inherit",
        }
      }
    >
      ← Go to Login
    </a>
    <div
      className="page-auth__logo"
    >
      <a
        href="/dashboard"
        onClick={[Function]}
        style={
          Object {
            "color": "inherit",
            "textDecoration": "inherit",
          }
        }
      >
        <div
          className="remarkably-logo"
        >
           
        </div>
      </a>
    </div>
  </div>
  <div
    className="page-auth__body"
  >
    <div
      className="account-form reset-password-success"
    >
      <div
        className="account-form__title"
      >
        Password has been reset!
      </div>
      <div
        className="account-form__subtitle"
      >
        Log into your account with your new password.
      </div>
      <div
        className="account-form__body"
      >
        <button
          className="button button--block button--uppercase button--primary goto-login__button"
          onClick={[Function]}
        >
          go to log in
        </button>
      </div>
    </div>
  </div>
  <div
    className="page-auth__footer"
  >
    © 2019 Remarkably. All Rights Reserved
  </div>
</div>
`;
